src/components/Photography/ExifOverlay.tsx

```diff
@@ -22,4 +28,28 @@ const ExifOverlay = styled.div`
   width: 100%;
 `
 
+interface Props {
+  image?: S3ImageAsset | any
+  isActive: boolean
+}
+
+const ExifOverlay = ({ image, isActive }: Props) => {
+  const EXIF = _.get(image, 'EXIF')
+  if (!EXIF) {
+    return null
+  }
+  const lensModel: string | undefined = mapLensModelExif(EXIF.LensModel || '')
+  const { FNumber, FocalLength, ISO, ShutterSpeedFraction } = EXIF
+  return (
+    <StyledExifOverlay isActive={isActive}>
+      {FocalLength ? `${FocalLength}mm, ` : null}
+      {ShutterSpeedFraction ? `${ShutterSpeedFraction}s, ` : null}
+      {FNumber ? `ƒ${FNumber}, ` : null}
+      {ISO ? `ISO ${ISO}` : null}
+      <br />
+      {lensModel}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)